README.md

```diff
@@ -54,20 +54,45 @@ LABEL authors='[\
         "email": "doe@john.com"\
     }\
 ]'
-LABEL docs='http://path.to.your.docs.com'
 LABEL company='{\
-  "about": "",\
+  "about": "brief description",\
+  "name": "Company/Person Name",\
+  "email": "email@company.com"\
+}'
+LABEL readme="https://raw.githubusercontent.com/username/repo/{tag}/README.md"
+LABEL links='{\
+  "website": "https://...",\
+  "support": "mailto:support@company.com",\
+  "documentation": "https://docs.company.com/cool-extension/",\
+}'
+LABEL type="example"
+LABEL tags='[\
+  "positioning",\
+  "navigation"\
+]'
 ```
 
  - `version` is the name of the current tag, which we expect to be a valid [semver](https://semver.org/).
  - `permissions`is a json file that follows the [Docker API payload for creating containers](https://docs.docker.com/engine/api/v1.41/#tag/Container/operation/ContainerCreate).
- - `docs` is a url for the documentation of your extension.
  - `authors` is a json list of authors of your extension
- - `company` is a json, which currently only contains an "about" section for a brief description about your company.
+ - `company` is a json with information about the maintainer responsible for providing new versions
+ - `readme` is a URL to a markdown-based README file for the extension
+ - `links` is a collection of additional useful/relevant links
```

Comment:
it does not say about the special case for docs and documentation

Reply:
I don't think that's particularly important to specify - the "links" label in general can contain arbitrary links that are decided on by the user, we'll most likely just be displaying them all as hyperlinks.